setup.py

```diff
@@ -96,7 +96,7 @@
     'Topic :: Software Development'
   ],
   install_requires = ['six>=1.11.0', 'subprocess32; python_version < "3"',
-      'python-dateutil==2.8.0', 'colorama>=0.3.9'],
+      'python-dateutil==2.8.0', 'PyKCS11==1.5.5; python_version > "3"', 'colorama>=0.3.9'],
```

Comment:
You're pinning here a version of PyKCS11 that is not available anymore on PyPI.

Reply:
Sorry, maybe I'm missing something, but in general we should be using `>=` not `==`, no?

Reply:
Sorry, I forgot to change that.

Reply:
BTW, you have the same version pinning in [ci-requirements.txt](https://github.com/secure-systems-lab/securesystemslib/pull/170/files?file-filters%5B%5D=.py&file-filters%5B%5D=.txt#diff-e513740081fff179cd93aa695e63334dR5) and [test-requirements.txt](https://github.com/secure-systems-lab/securesystemslib/pull/170/files?file-filters%5B%5D=.py&file-filters%5B%5D=.txt#diff-913fedaed3cb05fe362c0ceb35e2d827R8).

Reply:
Thanks for pointing this out. The various *-requirements.txt are due for revision (see #182).